notebooks/00_quick_start/wide_deep_movielens.ipynb

```diff
@@ -47,24 +47,23 @@
     "import sys\n",
```

Comment:
<p>You don't need MODEL_DIR = None</p><p>since you do</p><p>if MODEL_DIR is None:</p><p>&nbsp;&nbsp;tmp_dir = TemporaryDirectory()</p><p>&nbsp;&nbsp;MODEL_DIR = tmp_dir.name</p><p>afterwards.</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/686/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
I will get `name 'MODEL_DIR' is not defined` error if I don't define it. Right?

Reply:
I mean, you just need 
`tmp_dir = TemporaryDirectory()
MODEL_DIR = tmp_dir.name`
You don't need the if statement because there is no alternative.

Reply:
Ah, I got your point. MODEL_DIR can be passed as a parameter when the notebook got called by papermill. Usecase is to store model checkpoints.
Will add comments to make it clear.